### PR TITLE
Simplify support information

### PR DESCRIPTION
**Pull request by bahmady**
Proposal to simplify the information in the 'support' section. Much of this is duplicated unnecessarily from what's sent to partners when they sign on, some is incorrect or outdated, and some doesn't conform to our house style.

### CODE REVIEW

**Comment by bahmady** [comment]
Happy to put this in the test environment for review if it's easier but if you look in 'Files changed' and then 'View' it should show you the content more or less as it will be rendered live.

**Comment by bahmady** [comment]
Also something which occurs to me is that no mention is made of Bank Holidays?

File: source/documentation/16-support-contact-and-more-information.md [hunk 1]
```diff
@@ -31,77 +14,31 @@ Incident severity is defined below:
 | P3 | Significant | Users experiencing intermittent or degraded service due to platform issue | 1 business day or next working day | 2 business days |
 | P4 | Minor | Component failure that is not immediately service impacting | 2 business days | weekly |
```
Comment (bahmady) [inline source/documentation/16-support-contact-and-more-information.md]: @katiebates @heathd  can we add something here along the lines of "normal BAU functions" e.g. switching an account to live

**Comment by bahmady** [comment]
Should we add something along the lines of P4 being the priority level for a BAU request, e.g. switching an account to live?

**Comment by katiebates** [comment]
The simplification is a helpful change and I agree it's time to delete the sections about newsletters, press etc (or move them out of docs, to another product page). 

A few specific comments: 
We often tell partners that support is handled by the team, as a way to show that we have a direct feedback loop into product improvement, and the devs have a vested interest in ensuring the platform is operating robustly. Would it be useful/ appropriate to mention it here? It was alluded to in the last set of content: (e.g. "At all times, we have a dedicated Support Lead and Secondary (rotated weekly) who draw on additional support as required from other members of the team.. Additionally, a manager will act as the Support escalation route.")

In the Contact Us section, there is a reference to "support requests" but then to "support queries" - i don't think it's completely clear that a support request and query are the same thing. Also, I think it could be clearer that support requests come in via email to us, but we'll reply using the Zendesk ticketing system.

**Comment by bahmady** [comment]
> We often tell partners that support is handled by the team, as a way to show that we have a direct feedback loop into product improvement, and the devs have a vested interest in ensuring the platform is operating robustly. Would it be useful/ appropriate to mention it here? It was alluded to in the last set of content: (e.g. "At all times, we have a dedicated Support Lead and Secondary (rotated weekly) who draw on additional support as required from other members of the team.. Additionally, a manager will act as the Support escalation route.")

Here I will just entirely defer to you; if you would prefer that this be in this section that's your decision as far as I'm concerned. My opinion would be that it's not vital, simply because I'm accustomed to seeing users make use of sections like this to find out where and how to find support and when they can, and nothing much else. 

> In the Contact Us section, there is a reference to "support requests" but then to "support queries" - i don't think it's completely clear that a support request and query are the same thing. Also, I think it could be clearer that support requests come in via email to us, but we'll reply using the Zendesk ticketing system.

On the first point, agreed, this was a holdover that I should have caught. On the second point, I've tried to make it slightly clearer - I'm not sure if this level is what you were suggesting, but I can happily make even clearer still if not.

**Comment by katiebates** [comment]
I'm hesitant about spelling out switching to live specifically as a P4, as the SLA for P4 is 2 days. We can (and do) generate live accounts very quickly now, and want to emphasise how quickly we can get services taking payments.

**Comment by bahmady** [comment]
@katiebates I agree with you; I think that makes sense. Based on what else we discussed about this content, are you happy with it as-is or do you think it needs any further changes?

**Pull request merged**